.github/workflows/ci.yml

```diff
@@ -0,0 +1,121 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "v*"
+  pull_request:
+    branches:
+      - main
+  schedule:
+    # <minute [0,59]> <hour [0,23]> <day of the month [1,31]>
+    # <month of the year [1,12]> <day of the week [0,6]>
+    # https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07
+    # Run every Monday at 10:24:00 PST
+    # (Since these CRONs are used by a lot of people -
+    # let's be nice to the servers and schedule it _not_ on the hour)
+    - cron: "24 18 * * 1"
+  workflow_dispatch:
+
+jobs:
+  # Check that all files listed in manifest make it into build
+  check-manifest:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.x"
+      - run: pip install check-manifest && check-manifest
+  
+  # Check tests pass on multiple Python and OS combinations
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.8, 3.9]
+        os: [ubuntu-latest, macOS-latest]
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+    - uses: extractions/setup-just@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Install Conda Deps
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        auto-update-conda: true
+        python-version: ${{ matrix.python-version }}
+        activate-environment: anaconda-client-env
+        environment-file: ./environment.yml
+    - name: Install Dependencies
+      shell: bash -l {0}
+      run: |
+        python -m pip install --upgrade pip
+        just install
+    - name: Run Tests
+      shell: bash -l {0}
+      run: just test
+    - name: Upload Codecov
+      uses: codecov/codecov-action@v3
+
+  # Check linting, formating, types, etc.
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.9
+    - uses: extractions/setup-just@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install .[lint]
+    - name: Lint
+      run: just lint
+
+  # Publish to PyPI if test, lint, and manifest checks passed
+  publish:
+    if: "success() && startsWith(github.ref, 'refs/tags/')"
```

Comment:
why do you use the "" around this? I don't think you need to

Reply:
I think this is just a mistake in the cookiecutter template. I have no idea why it's there because it seems like the github docs would agree that no string wrapper is necessary :woman_shrugging: https://docs.github.com/en/actions/learn-github-actions/expressions#example-10

I will remove from cookiecutter and pull in that change.